Gemfile.lock

```diff
@@ -50,13 +50,14 @@ GEM
       cork
       nap
       open4 (~> 1.3)
-    codecov (0.1.15)
+    codecov (0.2.6)
```

Comment:
I got this error on the GitHub Actions CI:

> Fetching gem metadata from rubygems.org........
> Your bundle is locked to codecov (0.1.15), but that version could not be found
> in any of the sources listed in your Gemfile. If you haven't changed sources,
> that means the author of codecov (0.1.15) has removed it. You'll need to update
> your bundle to a version other than codecov (0.1.15) that hasn't been removed in
> order to install.

See [this build](https://github.com/wordpress-mobile/release-toolkit/pull/160/checks?check_run_id=1001022376).

This is bizarre because I can clearly see the gem for that version
on [RubyGems](https://rubygems.org/gems/codecov/versions)